task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace task_DEV_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter A Number :");
+            int inputNumber = int.Parse(Console.ReadLine());
+
+            if (inputNumber % 2 == 0 && inputNumber % 5 == 0)
+            {
+                Console.WriteLine("Tutti-Frutti");
+            }
+            else if (inputNumber % 2 == 0)
+            {
+                Console.WriteLine("Tutti");
+            }
+            else if (inputNumber % 5 == 0)
+            {
+                Console.WriteLine("Frutti");
+            }
+            else (!(inputNumber % 2 == 0) && !(inputNumber % 5 == 0))
```

Comment:
Does it work? When we use `else` block we don't need some conditions in such case. It looks just like
```
else
{
    // some code
}
```

Reply:
It would be useful for you. 
https://www.tutorialsteacher.com/csharp/csharp-if-else